x-pack/plugins/alerts/server/task_runner/task_runner.ts

```diff
@@ -166,24 +169,10 @@ export class TaskRunner {
       alertRawInstances,
       (rawAlertInstance) => new AlertInstance(rawAlertInstance)
     );
+    const originalAlertInstances = cloneDeep(alertInstances);
```

Comment:
we need the last set of alert instances (from the previous execution), so we can get the action group for "resolved" events